cppcache/integration-test-2/CMakeLists.txt

```diff
@@ -46,7 +47,7 @@ target_include_directories(integration-test-2
 
 target_link_libraries(integration-test-2
   PUBLIC
-    apache-geode
+    apache-geode-static
```

Comment:
Integration tests MUST use the dynamic library.

Reply:
Then we need separate unit tests.

Reply:
Integration tests should not test internal classes.

Reply:
I'm not testing an internal class, I'm dependent upon it. Efforts seem focused on migrating to this test suite and indications point toward abandoning the others, and there is misinformation: it was suggested to me to link to the static library, so not everyone is clear on the nature of this test suite.

We definitely need unit tests separate from integration tests. We should test the shared object as well as the static object and our internal classes. This much we agree. I'll discuss this with the team because it's clear to me not everyone is on the same page.